docs/programming-guide.md

```diff
@@ -1353,7 +1353,7 @@ data.map(x -> accum.add(x); f(x););
 <div data-lang="python"  markdown="1">
 {% highlight python %}
 accum = sc.accumulator(0)
-data.map(lambda x => acc.add(x); f(x))
+data.map(lambda x: acc.add(x) and f(x))
```

Comment:
@robertzk what do you think of that fix? + @ilganeli just because it looks like you created the example, although this bit may have been inadvertent copy and paste.